src/pages/payment-methods/_dp2p.js

```diff
@@ -0,0 +1,231 @@
+import React, { Fragment, useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { graphql, useStaticQuery } from 'gatsby'
+import device, { size } from 'themes/device'
+import { Text, Header, QueryImage } from 'components/elements'
+import { Flex } from 'components/containers'
+import { Button, LinkButton } from 'components/form'
+import { localize, Localize } from 'components/localization'
+import { isBrowser, deriv_dp2p_app_url, dp2p_google_play_url } from 'common/utility'
+import Checkmark from 'images/svg/checkmark.svg'
+
+const query = graphql`
+    query {
+        dp2p_platform: file(relativePath: { eq: "dp2p_platform.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const StyledContainer = styled.div`
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 4rem;
+
+    @media ${device.tabletS} {
+        flex-direction: column;
+    }
+`
+
+const ContentLeft = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+    max-width: 58.8rem;
+`
+
+const ContentRight = styled.div`
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    margin-left: 3.2rem;
+
+    @media ${device.tabletS} {
+        margin-left: 0;
+    }
+`
+
+const ItemTitle = styled.div`
+    display: flex;
+
+    img {
+        flex-shrink: 0;
+        width: 2.4rem;
+        height: 2.4rem;
+        margin: 0.6rem 0.8rem 0 0;
+        object-fit: contain;
+    }
+`
+
+const ImageWrapper = styled(Flex)`
+    max-width: 58.8rem;
+    width: 100%;
+    height: auto;
+    margin: 0.8rem 0 2.4rem;
+
+    div {
+        width: 100%;
+    }
+
+    @media ${device.tabletS} {
+        margin: 0.8rem auto 3.2rem;
+    }
+
+    @media ${device.mobileL} {
+        width: 100%;
+        height: 100%;
+    }
+`
+
+const ButtonWrapper = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+
+    @media ${device.mobileL} {
+        flex-direction: column;
+
+        button {
+            width: 100%;
+        }
+    }
+`
+
+const ButtonLearnMore = styled(LinkButton)`
+    height: 4rem;
+    @media ${device.mobileL} {
+        margin-bottom: 1.6rem;
```

Comment:
you can use px for media query

Reply:
@mustofa-binary what do you mean?

Reply:
rem is used for pre-set sizes, normally for sizing inside media query, we will use `px` instead of `rem` 🙏